boards/xtensa/intel_s1000_crb/doc/index.rst

```diff
@@ -125,6 +125,12 @@ software keys you have downloaded from `Tensilica Tools for Sue Creek`_
    follow what the license manager expects or create a dedicated ethernet device
    for this to work.
 
+   On the latest LTS ubuntu(18.04), you can install the license key manually with
+
+.. code-block:: console
+
+   cp sue-creek-SDK-license.dat <path to  SDK>/XtDevTools/install/tools/RF-2016.4-linux/XtensaTools/Tools/lic/license.dat".
```

Comment:
remove ". at the end.

Reply:
Hi Anas, it is addressed with commit 2b77c6b